msg/SensorOpticalFlow.msg

```diff
@@ -3,9 +3,9 @@ uint64 timestamp_sample
 
 uint32 device_id               # unique device ID for the sensor that does not change between power cycles
 
-float32[2] pixel_flow          # (radians) optical flow in radians where a positive value is produced by a RH rotation about the body axis
+float32[2] pixel_flow          # (radians) optical flow in radians where a negative value is produced by a RH rotation about the body axis
 
-float32[3] delta_angle         # (radians) accumulated gyro radians where a positive value is produced by a RH rotation about the body axis. Set to NaN if flow sensor does not have 3-axis gyro data.
+float32[3] delta_angle         # (radians) accumulated gyro radians where a negative value is produced by a RH rotation about the body axis. Set to NaN if flow sensor does not have 3-axis gyro data.
```

Comment:
Are you sure about the delta angle? This should just be an integral of the gyro data so it should be positive.

Reply:
delta angle tracks pixel_flow
![image](https://github.com/user-attachments/assets/b716b020-885c-4cde-b95b-e9fdb7483545)

Reply:
And how is it looking compared to `vehicle_angular_velocity`?

Reply:
vehicle_angular_velocity is downscaled
![image](https://github.com/user-attachments/assets/779d7ee6-f149-4340-a588-f4a5854d0abf)